### PR TITLE
abaqus 614 on sharc: rst, install and module files

### PR DESCRIPTION
**Pull request by dmrogers75**
Abaqus 6.14-2 on Sharc: 1 .rst, 1 install script and 1 module file.

### CODE REVIEW

**Comment by willfurnass** [comment]
Assign a reviewer if this is ready for review.

File: sharc/software/apps/abaqus.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
+Abaqus
+======
+
+.. sidebar:: Abaqus
+   
+   :Version: 6.14-2
+   :Dependencies: Intel compiler 15.0.7 and Foxit modules loaded. User subroutines need Intel compiler 2011 or above, GCC 4.1.2 or above. 
+   :URL: http://www.3ds.com/products-services/simulia/products/abaqus/ 
+   :Documentation: https://www.shef.ac.uk/wrgrid/software/abaqus
```
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: How up-to-date is the info at https://www.shef.ac.uk/wrgrid/software/abaqus?  Better to include all relevant reference info here?

File: sharc/software/apps/abaqus.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Usage
+-----
+
+Abaqus 6.14-2 can be activated using the module file::
+
+    module load apps/abaqus/614
+	
+Type ``abaqus cae`` to launch the Abaqus GUI during an interactive session with X Window support (e.g. an interactive ``qsh`` session).
+Type ``abaqus`` for the command line interface. Typing ``abaqus -help`` will display a list of usage options.
+The PDF viewer ``foxit`` can be launched to view the PDF documentation located at "/usr/local/packages/apps/abaqus/Documentation/docs/v6.14/pdf_books".
```
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Better formatted as fixed width text?

File: sharc/software/apps/abaqus.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+Usage
+-----
+
+Abaqus 6.14-2 can be activated using the module file::
+
+    module load apps/abaqus/614
+	
+Type ``abaqus cae`` to launch the Abaqus GUI during an interactive session with X Window support (e.g. an interactive ``qsh`` session).
```
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Prefix 'an' with 'from'?

File: sharc/software/apps/abaqus.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+
+Installation notes
+------------------
+
+Abaqus 6.14-2 was installed using the
+:download:`install_abaqus.sh </sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh>` script; the module
+file is
+:download:`614 </sharc/software/modulefiles/apps/abaqus/614>`. The binary installation was tested by launching ``abaqus cae`` and by using the above batch submission scripts.
+Abaqus at present does not run on more than one node when using MPI due to password-less ssh being disabled across nodes on ShARC.
```
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Still true?  Is there not a `catch_ssh` to complement the `catch_rsh` parallel environment startup method?

File: sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+# configure the variables for the build.
+# This script will then create the directories you need and download and unzip
+# the source in to the build dir.
+
+############################# Module Loads ###################################
+
+
+############################## Variable Setup ################################
+version=614
+prefix=/usr/local/packages/apps/abaqus/$version
```
Comment (willfurnass) [inline sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh]: Naming convention for installed binary packages is `apps/abaqus/$version/binary` (see Google Doc)

File: sharc/software/apps/abaqus.rst [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Abaqus is a software suite for Finite Element Analysis (FEA) developed by Dassault Systèmes.
+
+
+Usage
+-----
+
+Abaqus 6.14-2 can be activated using the module file::
+
+    module load apps/abaqus/614
```
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Might you need install a bug fix version before 6.15 comes out?   If so, what would the module file of that version be called?

File: sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh (hunk 5 above)
Comment (dmrogers75) [inline sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh]: Should I reinstall the code?

File: sharc/software/apps/abaqus.rst (hunk 6 above)
Comment (dmrogers75) [inline sharc/software/apps/abaqus.rst]: The */abaqus/614 etc. follows that of the installation on Iceberg. The latest version, yet to be installed on Sharc, is Abaqus 2017.

File: sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh (hunk 5 above)
Comment (willfurnass) [inline sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh]: If reinstalling would be difficult and people are already using the code then I imagine that outweighs the desire for consistency in this case :)

File: sharc/software/apps/abaqus.rst (hunk 4 above)
Comment (dmrogers75) [inline sharc/software/apps/abaqus.rst]: Sorry I don't understand. Is this to do with the "mpi-rsh" parallel environment set up for Ansys?

File: sharc/software/apps/abaqus.rst (hunk 6 above)
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Okay.

File: sharc/software/apps/abaqus.rst (hunk 4 above)
Comment (willfurnass) [inline sharc/software/apps/abaqus.rst]: Yes; I wondered if the same or a similar mechanism would help here?

File: sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh (hunk 5 above)
Comment (dmrogers75) [inline sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh]: Other codes have similair inconsistency (see email): Ansys and Ansys EM. If they are binaries then should be straightforward to copy/move? Would be good to be consistent. Any other binary installs without the "binary" in their path...

File: sharc/software/apps/abaqus.rst (hunk 4 above)
Comment (dmrogers75) [inline sharc/software/apps/abaqus.rst]: I have tried "mpi-rsh" but that doesn'et work. Anyway, Abaqus's internal version of PMPI does not support Omnipath.

File: sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh (hunk 5 above)
Comment (dmrogers75) [inline sharc/software/install_scripts/apps/abaqus/614/install_abaqus.sh]: New location wirh "binary" is specified.

File: sharc/software/modulefiles/apps/abaqus/614 [hunk 7]
```diff
@@ -0,0 +1,32 @@
+#%Module1.0#####################################################################
+##
+## ABAQUS 6.14 module file
```
Comment (willfurnass) [inline sharc/software/modulefiles/apps/abaqus/614]: No longer required?

Comment (dmrogers75) [inline sharc/software/modulefiles/apps/abaqus/614]: Yes, no longer requires as has been replaced.

Comment (dmrogers75) [inline sharc/software/modulefiles/apps/abaqus/614]: Replaced with sharc/software/modulefiles/apps/abaqus/6.14-2/binary

**Pull request merged**